tests/functional/ext/code_style/cs_consider_using_augmented_assign.py

```diff
@@ -61,17 +64,58 @@ def return_str() -> str:
 my_str = Unknown % my_str
 my_str = my_str % Unknown  # [consider-using-augmented-assign]
 
+# subtraction is anti-commutative
```

Comment:
```suggestion
# subtraction is not commutative
```

Reply:
This one is really anti-commutative, this is the one I extrapoled from :)